esp_serial_slave_link/essl.c

```diff
@@ -0,0 +1,240 @@
+// Copyright 2015-2019 Espressif Systems (Shanghai) PTE LTD
```

Comment:
Some files here have copyright in SPDX format while some older one. Suggest that we use same format for all files in this component.